docs/data/material/integrations/tailwindcss/tailwindcss-v4.md

```diff
@@ -125,6 +125,342 @@ Now you should see the autocomplete and syntax highlighting features when using
+  --shadow-2: var(--mui-shadows-2);
+  --shadow-3: var(--mui-shadows-3);
+  --shadow-4: var(--mui-shadows-4);
+  --shadow-5: var(--mui-shadows-5);
+  --shadow-6: var(--mui-shadows-6);
+  --shadow-7: var(--mui-shadows-7);
+  --shadow-8: var(--mui-shadows-8);
+  --shadow-9: var(--mui-shadows-9);
+  --shadow-10: var(--mui-shadows-10);
+  --shadow-11: var(--mui-shadows-11);
+  --shadow-12: var(--mui-shadows-12);
+  --shadow-13: var(--mui-shadows-13);
+  --shadow-14: var(--mui-shadows-14);
+  --shadow-15: var(--mui-shadows-15);
+  --shadow-16: var(--mui-shadows-16);
+  --shadow-17: var(--mui-shadows-17);
+  --shadow-18: var(--mui-shadows-18);
+  --shadow-19: var(--mui-shadows-19);
+  --shadow-20: var(--mui-shadows-20);
+  --shadow-21: var(--mui-shadows-21);
+  --shadow-22: var(--mui-shadows-22);
+  --shadow-23: var(--mui-shadows-23);
+  --shadow-24: var(--mui-shadows-24);
+
+  /* Material UI opacity */
+  --opacity-activated: calc(100% * var(--mui-palette-action-activatedOpacity));
+  --opacity-disabled: calc(100% * var(--mui-palette-action-disabledOpacity));
+  --opacity-focus: calc(100% * var(--mui-palette-action-focusOpacity));
+  --opacity-hover: calc(100% * var(--mui-palette-action-hoverOpacity));
+  --opacity-selected: calc(100% * var(--mui-palette-action-selectedOpacity));
+
+  /* Material UI overlays */
+  --overlay-1: var(--mui-overlays-1);
+  --overlay-2: var(--mui-overlays-2);
+  --overlay-3: var(--mui-overlays-3);
+  --overlay-4: var(--mui-overlays-4);
+  --overlay-5: var(--mui-overlays-5);
+  --overlay-6: var(--mui-overlays-6);
+  --overlay-7: var(--mui-overlays-7);
+  --overlay-8: var(--mui-overlays-8);
+  --overlay-9: var(--mui-overlays-9);
+  --overlay-10: var(--mui-overlays-10);
+  --overlay-11: var(--mui-overlays-11);
+  --overlay-12: var(--mui-overlays-12);
+  --overlay-13: var(--mui-overlays-13);
+  --overlay-14: var(--mui-overlays-14);
+  --overlay-15: var(--mui-overlays-15);
+  --overlay-16: var(--mui-overlays-16);
+  --overlay-17: var(--mui-overlays-17);
+  --overlay-18: var(--mui-overlays-18);
+  --overlay-19: var(--mui-overlays-19);
+  --overlay-20: var(--mui-overlays-20);
+  --overlay-21: var(--mui-overlays-21);
+  --overlay-22: var(--mui-overlays-22);
+  --overlay-23: var(--mui-overlays-23);
+  --overlay-24: var(--mui-overlays-24);
+}
+
+/* Material UI base styles */
+@layer base {
+  h1 {
+    font: var(--mui-font-h1);
+    letter-spacing: -0.01562em;
+  }
+  h2 {
+    font: var(--mui-font-h2);
+    letter-spacing: -0.00833em;
+  }
+  h3 {
+    font: var(--mui-font-h3);
+  }
+  h4 {
+    font: var(--mui-font-h4);
+    letter-spacing: 0.00735em;
+  }
+  h5 {
+    font: var(--mui-font-h5);
+  }
+  h6 {
+    font: var(--mui-font-h6);
+    letter-spacing: 0.0075em;
+  }
+  p {
+    font: var(--mui-font-body1);
+    letter-spacing: 0.00938em;
+  }
+  span {
+    font: var(--mui-font-body2);
+    letter-spacing: 0.01071em;
+  }
+}
+
+/* Material UI typography utilities */
+@utility typography-* {
+  font: --value(--font- *);
```

Comment:
This should have probably been:

```suggestion
  font: --value(--font-*);
```

per https://tailwindcss.com/docs/adding-custom-styles#matching-theme-values. 

Fixing it in #46685.